src/Symfony/Component/Form/Tests/AbstractLayoutTest.php

```diff
@@ -2730,4 +2731,38 @@ public function testButtonWithTranslationParameters()
 '
         );
     }
+
+    /**
+     * @dataProvider submitFormNoValidateProvider
+     */
+    public function testSubmitFormNoValidate(bool $attributeIsExpected, bool $validate)
```

Comment:
we could omit the `$attributeIsExpected` argument here and use `!$validate` instead below

Reply:
Indeed